packages/story-editor/src/components/videoTrim/trimmerComponents.js

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Slider from './slider';
+
+export const Menu = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: black;
+  height: 40px;
+  gap: 14px;
+`;
+
+export const Wrapper = styled.div`
+  position: relative;
+  flex-basis: ${({ pageWidth }) => pageWidth}px;
+  height: 36px;
+  border-radius: 4px;
+  border: 1px solid white;
```

Comment:
We have the `theme.colors.standard.white` color available as well.

Reply:
🙄

Reply:
Done in 6908ca2085.